maps/randomvaults/black_site_prism.dmm

```diff
@@ -0,0 +1,451 @@
+"fm" = (/obj/effect/decal/cleanable/blood/stattrack,/turf/simulated/floor/wood,/area/vault/black_site_prism)
+"fH" = (/obj/effect/landmark/corpse/syndicatesoldier{name = "Black Site Prism Operative"},/obj/item/weapon/shard,/obj/effect/decal/cleanable/blood{icon_state = "floor4"},/turf/unsimulated/floor{icon_state = "dark"},/area/vault/black_site_prism)
+"fP" = (/obj/machinery/chem_master,/turf/unsimulated/floor{icon_state = "dark"},/area/vault/black_site_prism)
+"gl" = (/obj/machinery/door/airlock/highsecurity{locked = 1; name = "Experimental Xenobiology"},/turf/unsimulated/floor{icon_state = "dark"},/area/vault/black_site_prism)
+"gF" = (/obj/item/organ/internal/eyes/grey,/obj/item/organ/internal/lungs/vox,/obj/item/xenos_claw,/obj/item/organ/internal/eyes/tajaran,/obj/structure/closet/crate/freezer/surgery,/turf/unsimulated/floor{icon_state = "dark"},/area/vault/black_site_prism)
+"gK" = (/obj/structure/table,/obj/item/stack/medical/bruise_pack,/obj/item/weapon/retractor{pixel_y = 6},/obj/item/weapon/scalpel,/obj/machinery/light{dir = 1},/turf/unsimulated/floor{icon_state = "dark"},/area/vault/black_site_prism)
+"gW" = (/obj/effect/decal/cleanable/blood/stattrack{dir = 10},/obj/effect/decal/cleanable/blood/splatter,/turf/unsimulated/floor{icon_state = "dark"},/area/vault/black_site_prism)
+"hg" = (/obj/structure/cable/yellow{d1 = 1; d2 = 2; icon_state = "1-2"},/turf/simulated/wall,/area/vault/black_site_prism)
+"hh" = (/obj/structure/grille,/obj/structure/window/reinforced{dir = 1},/obj/structure/window/reinforced,/obj/structure/window/reinforced{dir = 4; one_way = 1},/obj/structure/cable/yellow{d1 = 4; d2 = 8; icon_state = "4-8"},/turf/unsimulated/floor{icon_state = "dark"},/area/vault/black_site_prism)
+"hj" = (/obj/structure/window/reinforced{dir = 4},/turf/unsimulated/floor{icon_state = "dark"},/area/vault/black_site_prism)
+"hy" = (/obj/effect/decal/cleanable/blood/stattrack,/obj/effect/decal/cleanable/blood/drip,/obj/structure/grille/broken,/obj/item/weapon/shard,/obj/item/weapon/shard{icon_state = "small"},/obj/structure/cable/yellow{d1 = 4; d2 = 8; icon_state = "4-8"},/turf/simulated/floor/wood,/area/vault/black_site_prism)
+"hA" = (/obj/structure/closet/secure_closet/personal,/obj/abstract/map/spawner/security/armor,/turf/simulated/floor/carpet{icon_state = "carpetnoconnect"},/area/vault/black_site_prism)
+"hC" = (/obj/abstract/map/spawner/space/tools,/turf/unsimulated/floor{dir = 8; icon_state = "darkred"},/area/vault/black_site_prism)
+"hG" = (/obj/structure/window/reinforced{dir = 1},/turf/unsimulated/floor{dir = 9; icon_state = "carpetside"},/area/vault/black_site_prism)
+"hM" = (/obj/structure/safe/floor,/obj/item/weapon/disk/nuclear,/turf/unsimulated/floor{icon_state = "dark"},/area/vault/black_site_prism)
```

Comment:
Don't ever put a nuke disc in any other place than on station. This will despawn the disk existing on station and could many other problems. Think of something else here. You might have to code a nuclear disk variant specific to your vault, but you may not use the disk specific to the nuke ops gamemode.